### ISSUE TITLE
different faces but the program says they are the same

### ISSUE THREAD

**Issue by LorenceHernandez** [ageitgey/face_recognition#497]
* face_recognition version: Latest
* Python version: 3.6
* Operating System: windows server 2012

### Description

I've compared few images with different faces and same faces already and it returns correct output, but these ones returns true when compare but its clearly both different faces.
https://i.imgur.com/6pFeZq1.jpg
https://i.imgur.com/6KV1bqo.jpg

These are the pair of images that i tried to compare and it returns correct output:
pair1:
https://i.imgur.com/pLkc0ju.jpg
https://i.imgur.com/79ByRJU.jpg

pair2:
https://i.imgur.com/6KV1bqo.jpg
https://i.imgur.com/79ByRJU.jpg

pair3:
https://i.imgur.com/6KV1bqo.jpg
https://i.imgur.com/pLkc0ju.jpg

pair4:
https://i.imgur.com/6pFeZq1.jpg
https://i.imgur.com/pLkc0ju.jpg

pair5:
https://i.imgur.com/6pFeZq1.jpg
https://i.imgur.com/79ByRJU.jpg
### What I Did

```

import face_recognition

pic1 = 'pic1.jpg'
pic2 = 'pic2.jpg'
candidate_image_file = face_recognition.load_image_file(pic1)
user_image_file = face_recognition.load_image_file(pic2)
candidate_encoding = face_recognition.face_encodings(candidate_image_file)[0]
user_image_encoding = face_recognition.face_encodings(user_image_file)[0]
results = face_recognition.compare_faces([candidate_encoding], user_image_encoding)

print(results) ''' prints [True] when it should be false [False]
	

```

**Comment by tlugger** [comment]
I would decrease the tolerance used by `face_recognition.compare_faces`. By default the `compare_faces` method uses a tolerance of `0.6` to compare encodings. I've found that I get fewer false positives by bumping that down to `0.5`

New results line:
```
results = face_recognition.compare_faces([candidate_encoding], user_image_encoding, 0.5)
```

Hope this helps!

**Comment by vrushang25** [comment]
yes please decrease tolerance value you will get best accuracy for face recognition
